utils/drop.ts

```diff
@@ -31,4 +31,4 @@ export const AHP_GENERATIVE_DROPS = [
 export const AHP_POPULAR_DROP_COLLECTIONS = [...AHP_GENERATIVE_DROPS]
 
 export const FALLBACK_DROP_COLLECTION_MAX = 300
-export const FUTURE_DROP_DATE = new Date(2023, 11, 28, 17)
+export const FUTURE_DROP_DATE = new Date(Date.UTC(23, 11, 28, 18, 0, 0))
```

Comment:
```suggestion
export const FUTURE_DROP_DATE = new Date(Date.UTC(2023, 11, 28, 18, 0, 0))
```

Because it does not work
<img width="230" alt="image" src="https://github.com/kodadot/nft-gallery/assets/31397967/c2db08b4-e8d8-464c-abe6-2e0910d8f2e0">

Reply:
Stole it from MDN .) 😭